spec/support/landing_folder_support/file_objects/base_pdf_file.rb

```diff
@@ -8,8 +9,17 @@ class BasePdfFile < Base
         include EtApi::Test::I18n
         include ::RSpec::Matchers
 
+        def initialize(tempfile, form = nil, template:, lookup_root:)
```

Comment:
Part of pdf test code refactor - much more shared between claim and response pdf files